config/registry_image_pruner/cronjob.yaml

```diff
@@ -33,7 +33,18 @@ spec:
             volumeMounts:
               - name: image-pruner-volume
                 mountPath: /image-pruner
+            resources:
+              limits:
+                cpu: 500m
+                memory: 128Mi
+              requests:
+                cpu: 5m
+                memory: 64Mi
```

Comment:
@sbose78 @Michkov I opened this PR to address all kube-linter issues in this project but I have no clue how much cpu/memory to request and limit. I did put some number to open the PR but let me know what number I should put here

Reply:
@mmorhun ^^

Reply:
Hello @hugares. Sorry for the delay.
I've investigated a bit the values you are setting. For a regular run, the job consumed `105m` CPU and `82MiB` RAM. So, my suggestion to have `150m`/`500m` CPU and `128Mi`/`512Mi` RAM.

Reply:
Done